tests/functional/test_install.py

```diff
@@ -271,6 +271,7 @@ def test_install_from_local_directory(script, data):
     assert egg_info_folder in result.files_created, str(result)
 
 
+@pytest.mark.skipif("sys.version_info[:2] == (3, 3)")
```

Comment:
Damn! I meant to gating on version to check error message behaviour seems weird.